src/test/codewhisperer/util/crossFileContextUtil.test.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+import * as semver from 'semver'
+import { getRelevantCrossFiles } from '../../../codewhisperer/util/supplementalContext/crossFileContextUtil'
+import { shuffleList, closeAllEditors, toFile, assertTabSize } from '../../testUtil'
+import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
+import { normalize } from '../../../shared/utilities/pathUtils'
+
+// TODO: make it a util function inside testUtil.ts
+let tempFolder: string
+
+async function openATextEditorWithText(fileText: string, fileName: string): Promise<vscode.TextEditor> {
+    const completeFilePath = path.join(tempFolder, fileName)
+    toFile(fileText, completeFilePath)
+
+    const textDocument = await vscode.workspace.openTextDocument(completeFilePath)
+
+    return await vscode.window.showTextDocument(textDocument, { preview: false })
+}
+
+describe('getRelevantFiles', async function () {
+    before(async function () {
+        this.timeout(600000)
+    })
+
+    beforeEach(async function () {
+        tempFolder = await makeTemporaryToolkitFolder()
+    })
+
+    afterEach(async function () {
+        await closeAllEditors()
+        await assertTabSize(0)
+        await fs.remove(tempFolder)
```

Comment:
consider using `tryRemoveFolder` https://github.com/aws/aws-toolkit-vscode/blob/4c9713be84882cdad8728f364b02f8546a202b72/src/shared/filesystemUtilities.ts#L96

even if it doesn't fix the issue, it's better to use the shared library code so that callers get improvements when we improve the library code.

Reply:
gotcha let me try it